app-editors/emacs/emacs-28.1.91-r1.ebuild

```diff
@@ -317,6 +316,34 @@ src_compile() {
 	emake
 }
 
+src_test() {
+	# List .el test files with a comment above listing the exact
+	# subtests which caused failure. Elements should begin with a %.
+	# e.g. %lisp/gnus/mml-sec-tests.el.
+	local excluded_tests=(
+		# Reason: not yet known
+		# mml-secure-en-decrypt-{1,2,3,4}
+		# mml-secure-find-usable-keys-{1,2}
+		# mml-secure-key-checks
+		# mml-secure-select-preferred-keys-4
+		# mml-secure-sign-verify-1
+		%lisp/gnus/mml-sec-tests.el
+
+		# Reason: race condition
+		# Looks like it should be fixed in 29.x at least:
+		# https://debbugs.gnu.org/cgi/bugreport.cgi?bug=55706
+		# files-tests-file-name-non-special-file-in-directory-p
+		%lisp/files-tests.el
+	)
+
+	# See test/README for possible options
+	emake \
+		EMACS_TEST_VERBOSE=1 \
+		EXCLUDE_TESTS="${excluded_tests[*]}" \
+		TEST_BACKTRACE_LINE_LENGTH=100 \
```

Comment:
What is wrong with the default (70 AFAICS)?

Reply:
I had useful backtraces truncated repeatedly (they don't continue onto the next line) :(

Reply:
That may also happen at 100. Let's make it effectively infinite, like 9999?

Reply:
Ah, nice, I didn't realise it supported `nil`. Thank you, let me try that.